safe_transaction_service/history/serializers.py

```diff
@@ -982,9 +970,10 @@ class AllTransactionsSchemaSerializer(serializers.Serializer):
 # Deprecated ---------------------------------------------------------------
 
 
-class SafeDelegateDeleteSerializer(serializers.Serializer):
+class SafeDelegateDeleteDeprecatedSerializer(serializers.Serializer):
```

Comment:
Other deprecated classes were keeping the class name but with a comment that the class is deprecated, are you aware about that this class is renamed including the Deprecated tag?

Reply:
Updated the name, it had been forgotten. As for the reasons, it was in [this PR](https://github.com/safe-global/safe-transaction-service/pull/409), but I don't know why.